test/cpp/common/timer_test.cc

```diff
@@ -157,17 +157,69 @@ TEST_F(TimerTest, CancelSomeTimers) {
 
 // Enable the following test after
 // https://github.com/grpc/grpc/issues/20049 has been fixed.
-#if 0
-TEST_F(TimerTest, TimerNotCanceled) {
+TEST_F(TimerTest, DISABLED_TimerNotCanceled) {
   grpc_core::ExecCtx exec_ctx;
   grpc_timer timer;
   grpc_timer_init(&timer, 10000,
-                  GRPC_CLOSURE_CREATE(
-                      [](void*, grpc_error*) {
-                      },
-                      nullptr, grpc_schedule_on_exec_ctx));
+                  GRPC_CLOSURE_CREATE([](void*, grpc_error*) {}, nullptr,
+                                      grpc_schedule_on_exec_ctx));
+}
+
+// Enable the following test after
+// https://github.com/grpc/grpc/issues/20064 has been fixed.
+TEST_F(TimerTest, DISABLED_CancelRace) {
+  MAYBE_SKIP_TEST;
+  grpc_core::ExecCtx exec_ctx;
+  const int kNumTimers = 10;
+  grpc_timer timers[kNumTimers];
+  for (int i = 0; i < kNumTimers; ++i) {
+    grpc_timer* arg = nullptr;
+    if (i != 0) {
+      arg = &timers[i - 1];
+    }
+    grpc_timer_init(&timers[i], 100,
```

Comment:
The second param of this function is deadline. Do you intent to use `100` as deadline here? Or timeout?

Reply:
Good point, it should be `grpc_core::ExecCtx::Get()->Now() + timeout`. Fixed.